src/styles.scss

```diff
@@ -851,6 +851,20 @@ body {
   vertical-align: middle;
 }
 
+// Set various font attributes
+.weight-500 {
+  font-weight: 500;
+}
+
+.size-12 {
```

Comment:
PR checklist, need a comment for this.

From what I can tell, the mockups (not including headings) only use 14px (default) or 12px. There's no 13, 11.9, etc. If true and in order to maintain consistency, the comment should mention the use case for this.  Would hopefully prevent others from using `<small`, or font-size: small, font-size: smaller, and we would use this else where instead.